packages/web/src/components/rewards/modals/VipDiscordModal.tsx

```diff
@@ -24,7 +22,7 @@ const messages = {
 }
 
 export const VipDiscordModal = () => {
-  const discordCode = useSelector(getDiscordCode)
+  const { data: discordCode } = useDiscordCode('audio')
```

Comment:
why do we use `audio` here and `TOKEN_LISTING_MAP.AUDIO.symbol` in `packages/mobile/src/components/vip-discord-drawer/VipDiscordDrawer.tsx`? also, shouldn't this value be backend driven?